app/tests/spec/views/permissions.js

```diff
@@ -122,10 +122,10 @@ function (chai, $, sinon, p, View, AuthErrors, Metrics, FxaClient,
       it('renders relier info', function () {
         return initView('sign_up')
           .then(function () {
-            assert(!! view.$('#fxa-permissions-header').text().match(SERVICE_NAME),
-              'service name shows in header');
+            assert(!! view.$('p').text().match(SERVICE_NAME),
```

Comment:
Another thing - you can use `assert.notInclude` here - http://chaijs.com/api/assert/#notInclude